Source/Initialization/PlasmaInjector.H

```diff
@@ -62,7 +62,8 @@ public:
     amrex::Real getMass () {return mass;}
     PhysicalSpecies getPhysicalSpecies() const {return physical_species;}
 
-    bool doInjection () const noexcept { return h_inj_pos != nullptr;}
+    // bool: whether the initial injection of particles should be done
```

Comment:
Could we add this clarification to the inline comment, for those less familiar with this class?
```suggestion
    // bool: whether the initial injection of particles should be done
    // (if surface_flux is true, AddPlasmaFlux will be called instead of AddPlasma)
```

Reply:
Thanks for your review @EZoni. I'll add a comment here, though somewhat different.